labs/apis/postman_lab/Intro-to-postman

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
Q-2: This file and collection are not named in accordance with the instructions.